releasenotes/notes/57600.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
```

Comment:
@jaellio is this a featue? if so, we didn't backport feature.

Reply:
It's required for this fix - https://github.com/istio/istio/pull/57456

Reply:
We could wait until 1.28 if that is preferred since this is changing installation behavior

Reply:
Could you request an exception from TOC?